docs/data/base/components/modal/NestedModal.js

```diff
@@ -224,15 +223,16 @@ const ModalButton = styled(Button)(
   }
 
   &:active {
-    background: ${theme.palette.mode === 'dark' ? grey[700] : grey[100]};
+    background-color: ${blue[700]};
+    box-shadow: none;
   }
 
   &:focus-visible {
     box-shadow: 0 0 0 4px ${theme.palette.mode === 'dark' ? blue[300] : blue[200]};
     outline: none;
   }
 
-  &.disabled {
+  &:disabled {
```

Comment:
Same question as above

Reply:
The `:disabled` and `:active` pseudoclasses are a bit different to `:focus-visible`. They don't exist to work around browser support issues, but add functionality to non-interactive elements. For example, if a `<span>` is used as a button, it wouldn't have the `:disabled` state, and `:active` would work a bit differently. In case of this demos, it does not matter, as we're using a native button, but in general it's safer to use the classes than pseudoclasses (as they work everywhere).